i18n/shared/src/main/scala/io/udash/i18n/RemoteTranslationRPC.scala

```diff
@@ -21,5 +22,4 @@ trait RemoteTranslationRPC {
   /** Returns map of translations and bundle hash. If `oldHash` is not outdated, this Future will contain None. */
   def loadTranslationsForLang(lang: Lang, oldHash: BundleHash): Future[Option[Bundle]]
 }
-
-
+object RemoteTranslationRPC extends RPCTypeClasses[DefaultServerUdashRPCFramework.type, RemoteTranslationRPC]
```

Comment:
We cannot assume that someone's using `DefaultServerUdashRPCFramework`. I think we cannot provide `RPCTypeClasses` for `RemoteTranslationRPC` and we can add a note in Udash Guide.